azurerm/internal/services/containers/resource_arm_kubernetes_cluster.go

```diff
@@ -237,6 +237,28 @@ func resourceArmKubernetesCluster() *schema.Resource {
 				},
 			},
 
+			"service_principal": {
```

Comment:
I think you'll want to revert this change?

Reply:
nah this is just a minor regrouping from where the `service_principal` block went from required -> optional -> required in the last PR and I didn't want travis to block the release - so this should be fine 👍